fabric-api-catalog/build.gradle

```
@@ -0,0 +1,57 @@
+plugins {
+	id 'version-catalog'
+}
+
+version = rootProject.version
+
+publishing.publications {
+	register('mavenJava', MavenPublication) {
+		from(components['versionCatalog'])
+	}
+}
+
+tasks.withType(GenerateModuleMetadata) {
+	// todo: RemoteSignJar fails when this is false (as set by parent build script)
+	enabled = true
+}
+
+// Avoid configuration ordering issues by creating the catalog entries during task execution time
+tasks.register('configureCatalog') {
+	doFirst {
+		doConfigureCatalog()
+	}
+}
+tasks.generateCatalogAsToml {
```

This should be enough in Groovy DSL buildscripts:
```suggestion
generateCatalogAsToml {
```
Or lazily:
```suggestion
tasks.named('generateCatalogAsToml') {
```